### DISCUSSION TITLE
camel-karavan-oidc is outdated. Is it discontinued?

### DISCUSSION THREAD

**Discussion by arheom** [apache/camel-karavan#1127] [General]
I see inside the git action, the camel-karavan-oidc is commented out and the last release of it was in Sep last year. Is it discontinued? We are using the oidc version, which means currently we need to fork the code and sync our releases.

If it is not discontinued, it would be great to allow that release to also get publish in the same time with the public release, so we could connect directly to it.

**Comment by mgubaidullin** [comment]
https://github.com/apache/camel-karavan/blob/main/.github/workflows/app-oidc.yml

**Reply by arheom** [reply]
Thanks @mgubaidullin . Could you also update the actions with the multi-arch update? https://github.com/apache/camel-karavan/commit/0e89fc64c9360c5cab9654276430860150211123 . I think they were overwritten, and would be great to be added to both action flows.

Thanks!

**Reply by mgubaidullin** [reply]
I revert it back because it should not be jib build but docker. 
Need your help with that. 
https://github.com/apache/camel-karavan/issues/1128

**Reply by arheom** [reply]
thanks! will look into it. when is it planned to have 4.4.0? next week I am off and I will look after that. Hope wont affect the schedule.

**Reply by mgubaidullin** [reply]
Apache Camel Framework 4.4.0 is on the vote. So it should be released 🤞 next week.
Karavan is generated from Camel DSL and Components metadata, so we will start generate it next week.
The week after we will prepare Karavan release. 
Take your time and come back full of energy )
